frontend/Lighting.jsx

```diff
@@ -67,43 +69,78 @@ function Lighting() {
   }), [lightOn, lightColor, lightIntensity]));
 
   return (
-    <View style={styles.container}>
-      <Text>
-        {'Turn Light On\n\n'}
-      </Text>
-      <Switch
-        trackColor={{ false: '00FF00', true: 'FFFFFF' }}
-        onValueChange={(value) => setLightOn(value)}
-        value={lightOn}
-        style={{ transform: [{ scaleX: 2 }, { scaleY: 2 }] }}
-      />
-      <Text>
-        {`\n\n\n\nIntensity of Light: ${Math.round(lightIntensity)}`}
-      </Text>
-      <Slider
-        minimumValue={0}
-        maximumValue={3}
-        disabled={!lightOn}
-        value={lightIntensity}
-        onValueChange={(value) => setLightIntensity(value)}
-        style={{ width: 160, height: 50 }}
-        minimumTrackTintColor="#ffd700"
-        maximumTrackTintColor="#fafad2"
-      />
-      <Text>
+    <LinearGradient
+      colors={['#F9F6FF', '#CFDFF7']}
+      style={{
+        flex: 1,
+        padding: 10,
+        paddingTop: 80,
+        alignItems: 'center',
+      }}
+    >
+      <View style={{ flexDirection: 'row', justifyContent: 'space-between', marginBottom: 30 }}>
+        <LightingTitle>Lighting</LightingTitle>
+        <Switch
+          trackColor={{ false: '00FF00', true: 'FFFFFF' }}
+          onValueChange={(value) => setLightOn(value)}
+          value={lightOn}
+          style={{ transform: [{ scaleX: 1.4 }, { scaleY: 1.4 }] }}
+        />
+      </View>
+      <Subheading>
+        {`Intensity of Light: ${Math.round(lightIntensity) * 10}%`}
```

Comment:
If we are gonna display intensity as a percentage, maybe we can display to the ones instead of tens.
```suggestion
        {`Intensity of Light: ${lightIntensity.toFixed(1) * 10}%`}
```

Reply:
Alternatively we can make the slider go from 0 to 100 and round that instead